weatherAplication/Model/StorageService.swift

```diff
@@ -0,0 +1,35 @@
+import Foundation
+
+final class StorageService {
+    private let lastSearchesKey = "lastSearches"
+
+    static let sharedInstance = StorageService()
+
+    func getCitiesInfo(failure: @escaping ((Error) -> Void)) -> [CityInfo] {
+        var citiesInfoArray: [CityInfo] = []
+        if let data = UserDefaults.standard.data(forKey: lastSearchesKey) {
+            do {
+                citiesInfoArray = try PropertyListDecoder().decode([CityInfo].self, from: data)
```

Comment:
PropertyListEncoder - nie podoba mi się ta nazwa, powinna być bardziej ogólna bo chyba nie wykonuje ta klasa jakichś wyspecyfikowanych rzeczy (innych niż porostu dekodowanie)

Reply:
to jest klasa z foundation https://developer.apple.com/documentation/foundation/propertylistdecoder więc nie bardzo mam jak zmienić jej nazwę (chyba że jakoś się da a ja o tym nie wiem)